### PR TITLE
Chris/fix template python source

### PR DESCRIPTION
**Pull request by ChristopheDuong**
## What
Fixing more paths from airbyte-integration moves and making the python-source template work again

## How
I also added the lint & formatting config file as a start and run black on the python code

### CODE REVIEW

**Review by michel-tricot — APPROVED** [review]
Can we have more columns (100-120) or is it something hardcoded by one of the linter?

File: airbyte-integrations/connector-templates/python-source/.flake8 [hunk 1]
```diff
@@ -0,0 +1,6 @@
+[flake8]
```
Comment (jrhizor) [inline airbyte-integrations/connector-templates/python-source/.flake8:1]: these configs should probably not be part of this pr

**Review by cgardens — APPROVED** [review]
